.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "type": "shell",
+            "command": "rm -rf Debug && mkdir Debug && cd Debug && cmake -DCMAKE_BUILD_TYPE=Debug .. && make"
```

Comment:
I don't think we should be adding use of the `rm -rf` command in a checked-in file. It is too silently "destructive" :p

Reply:
this file is required for enabling debugging on VScode.
If we want people to be able to download and debug using vscode, they need this (or figure it out how to create one)